openghg/store/base/_base.py

```diff
@@ -40,6 +40,10 @@ def __init__(self, bucket: str) -> None:
         self._bucket = bucket
         self._metakey = ""
         self._metastore = load_metastore(bucket=bucket, key=self.metakey())
+        # Do I store the changes we've made to the object store in __exit__
```

Comment:
Is this a question to be addressed later? May want to add a TODO at the start if so?

Reply:
Ah I started off with that but forgot to remove it. I've deleted it now.